oshi-core/src/main/java/oshi/software/os/OperatingSystem.java

```diff
@@ -131,7 +238,10 @@
      *         specified. The list may contain null elements or processes with a
      *         state of {@link OSProcess.State#INVALID} if a process terminates
      *         during iteration.
+     * @deprecated Use {@link #getProcesses(Predicate, Comparator, int)} with
+     *             sorting constants from {@link ProcessSorting}.
      */
+    @Deprecated
     List<OSProcess> getProcesses(int limit, ProcessSort sort);
```

Comment:
perhaps this can be a default method, so implementers can forget about it

Reply:
Thanks, great suggestion.  Probably can be applied throughout other classes eventually.

Reply:
Well I tried to do this, but the implementation uses a private helper method common to the two deprecated methods (to convert the `ProcessSort` enum to the appropriate constant).  And private methods in interfaces is a JDK9+ feature.

So for now I'll just leave the implementation as-is.

Reply:
Or, actually, I had a nested class with the new constants which is a perfect place for a temporary private helper method to use those constants. Problem solved.